cmd/kogito/command/converter/kafkameta_converter_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
```

Comment:
If the file is new, why `2019`? You can just create a blank go file, the `make test|addheaders` will add the headers for you. :)

There are other files with the same problem, I won't point each one to not pollute your PR.

Reply:
done